andela/mmdp_site
# #MMDP-616 Fixes thematic pillar count to show on selected LGAs

**What does this PR do?**

It fixes thematic pillar count to display on the state level map section of the Content Management System

**description of Task to be completed?**

- Show the number of thematic pillars within each selected LGA map that has been clicked on the state level map. This will tell how many thematic pillars are within a specific LGA from one view.

**How should this be manually tested?**

- git fetch
- git checkout to this branch `bg-fix-thematicPillarCount-MMDP-616`
- click on the CMS on the coordination matrix, state level. Click view next to each LGA to see the thematic pillars on that LGA

**Any background context you want to provide?**

None

**What are the relevant Jira issues?**

[MMDP-616](https://viisaus.atlassian.net/browse/MMDP-616)

**Screenshots (if appropriate)**

<img width="1680" alt="Screenshot 2019-06-04 at 10 23 11" src="https://user-images.githubusercontent.com/23031493/58861228-909eb200-86b6-11e9-8024-430058203027.png">
